packages/components/src/components/search-box/search-box.jsx

```diff
@@ -6,6 +6,7 @@ import Icon from '../icon';
 import Input from '../input';
 
 const SearchBox = ({ className, onClear, onSearch, placeholder }) => {
+    let typing_timer;
```

Comment:
let's store the timeout id in a `React.Ref()`, so that it's available all the time, e.g. between component updates :)